### PR TITLE
feat: Add sample using Next JS (React + TS)

### PR DESCRIPTION
**Pull request by teddyward**
Three things here:

- Add another hello world add-on. This is the same as https://github.com/googleworkspace/meet/pull/13, but done in React. You can see it live at https://googleworkspace.github.io/meet/hello-world-next-js/sidepanel
- Update the push rules so that all sample add-ons are pushed
- Fix a couple things, after publishing the old hello world add-on privately

I installed this add-on to my corp account and everything seems to work great. I can send the screen recording if ppl want :) working on getting marketplace review...

Happy to split this up if people want.

I gave up on Vite. see https://github.com/googleworkspace/meet/tree/teddy_sample-vite (plus some internal bugs) for my failures

### CODE REVIEW

File: addons-web-sdk/samples/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for sample in ./*/
```
Comment (clayplumridge) [inline addons-web-sdk/samples/build.sh:1]: Could get around this using workspace commands and `concurrently` but this is probably fine. The npm version of these commands is also less nice than the yarn version.

File: addons-web-sdk/samples/hello-world/package-lock.json
Comment (clayplumridge) [inline addons-web-sdk/samples/hello-world/package-lock.json:1]: Should this package-lock also be excluded?

**Review by clayplumridge — DISMISSED** [review]

**Review by benediktbrandt — APPROVED** [review]

File: addons-web-sdk/samples/build.sh (hunk 1 above)
Comment (teddyward) [reply addons-web-sdk/samples/build.sh]: OH! I didn't really understand workspaces. Thanks for this! I didn't use concurrently but now it's much cleaner -- just calls `npm run build --workspaces`

File: addons-web-sdk/samples/hello-world/package-lock.json
Comment (teddyward) [reply addons-web-sdk/samples/hello-world/package-lock.json:1]: yes and the one in the root should be included! thanks for that catch.

**Comment by teddyward** [comment]
thanks Clay! all set now.

**Review by clayplumridge — DISMISSED** [review]

**Review by benediktbrandt — APPROVED** [review]

**Pull request merged**